### PR TITLE
Refactored ToggleSwitch Default Native XAML Styles and added documentation.

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
Refactored ToggleSwitch Default Native XAML styles, BindableSwitchCompat.Android, BindableUISwitch.iOS and added related documentation.

## PR Type
What kind of change does this PR introduce?

- Feature

- Refactoring (no functional changes, no api changes)

- Documentation content changes

## What is the current behavior?
- Current ToggleSwitch default native styles were missing crucial properties in order to be able to style them correctly. 
- Missing documentation related to the ToggleSwitch Control.

## What is the new behavior?
- Added ToogleSwitch documentation in Uno/Doc/Controls.
- Added new properties for ToggleSwitch Default Native Styles.
  [iOS] For BindableUISwitch : TintColorBrush property was added to be able to tint the outline of the 
  switch when it is turned off.
  [Android] For BindableSwitchCompat : 
   - Text property was added in order to change the ToggleSwitch label.
   - TextColor property was added in order to change the ToggleSwitch label color.
   - ThumbTint property was added in order to change the Thumb color.
   - TrackTint property was added in order to change the Track color.

**Breaking changes**
- Refactored ToggleSwitch Default Native XAML Styles. (cf. 'NativeDefaultToggleSwitch' styles in Generic.Native.xaml)
  [iOS] For BindableUISwitch : Background property was changed for OnTintColorBrush and Foreground 
  property for ThumbTintColorBrush.
  [Android] BindableSwitch was renamed BindableSwitchCompat in order to avoid confusion with the 
  Switch control.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_workitems/edit/138540

### CODE REVIEW

File: src/Uno.UI/Controls/BindableSwitchCompat.Android.cs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+			// E / AndroidRuntime(6313): 	at android.support.v7.widget.SwitchCompat.onMeasure(SwitchCompat.java:526)
+			// E / AndroidRuntime(6313): 	at android.view.View.measure(View.java:17547)
+
+			TextOn = "";
+			TextOff = "";
+		}
+
+		#region TextColor DependencyProperty
+
+		//The color used to change the text header foreground.
```
Comment (jeromelaban) [inline src/Uno.UI/Controls/BindableSwitchCompat.Android.cs]: Use the xml comment syntax

File: src/Uno.UI/Controls/BindableSwitchCompat.Android.cs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+			set { this.SetValue(TextColorProperty, value); }
+		}
+
+		public static readonly DependencyProperty TextColorProperty =
+			DependencyProperty.Register("TextColor", typeof(Brush), typeof(BindableSwitchCompat), new PropertyMetadata(null, (s, e) => ((BindableSwitchCompat)s).OnTextColorChanged((Brush)e.NewValue)));
+
+		private void OnTextColorChanged(Brush newValue)
+		{
+			var asColorBrush = newValue as SolidColorBrush;
+			if (asColorBrush != null)
```
Comment (jeromelaban) [inline src/Uno.UI/Controls/BindableSwitchCompat.Android.cs]:
```suggestion
			if (newValue is SolidColorBrush asColorBrush)
```

File: src/Uno.UI/Controls/BindableSwitchCompat.Android.cs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+		public BindableSwitchCompat(Android.Content.Context context, IAttributeSet attrs)
+			: base(context, attrs)
+		{
+			InitializeBinder();
+		}
+
+		public BindableSwitchCompat(Android.Content.Context context, IAttributeSet attrs, int defStyle)
+			: base(context, attrs, defStyle)
+		{
+			InitializeBinder();
```
Comment (jeromelaban) [inline src/Uno.UI/Controls/BindableSwitchCompat.Android.cs]: Not needed
```suggestion
```

File: src/Uno.UI/Controls/BindableSwitchCompat.Android.cs [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+		private void OnTextChange(object sender, TextChangedEventArgs e)
+		{
+			SetBindingValue(Text, "Text");
+		}
+
+		public BindableSwitchCompat(Android.Content.Context context, IAttributeSet attrs)
+			: base(context, attrs)
+		{
+			InitializeBinder();
```
Comment (jeromelaban) [inline src/Uno.UI/Controls/BindableSwitchCompat.Android.cs]:
```suggestion
```

File: src/Uno.UI/Controls/BindableUISwitch.iOS.cs [hunk 5]
```diff
@@ -54,17 +55,19 @@ public BindableUISwitch(IntPtr handle)
...
-			get { return (Brush)this.GetValue(ForegroundProperty); }
-			set { this.SetValue(ForegroundProperty, value); }
+			get { return (Brush)this.GetValue(ThumbTintColorBrushProperty); }
+			set { this.SetValue(ThumbTintColorBrushProperty, value); }
 		}
 
-		// Using a DependencyProperty as the backing store for Background.  This enables animation, styling, binding, etc...
-		public static readonly DependencyProperty ForegroundProperty =
-			DependencyProperty.Register("Foreground", typeof(Brush), typeof(BindableUISwitch), new PropertyMetadata(null, (s, e) => ((BindableUISwitch)s).OnForegroundChanged((Brush)e.NewValue)));
+		public static readonly DependencyProperty ThumbTintColorBrushProperty =
```
Comment (jeromelaban) [inline src/Uno.UI/Controls/BindableUISwitch.iOS.cs]: This is a binary breaking change, keep the `ForegroundProperty` and make it an alias to the other property, same for Background.

Comment (agneszitte-nventive) [reply src/Uno.UI/Controls/BindableUISwitch.iOS.cs]: Done Thanks !

**Pull request merged**